docs/syzbot.md

```diff
@@ -164,8 +164,20 @@ so please treat the results with understanding. You may consult the provided
 Suggestions and patches that improve bisection quality for common cases are
 [welcome](https://github.com/google/syzkaller/issues/1051).
 
+Syzbot supports cause bisection(find the commit that introduces a bug) and fix bisection(find the commit that fixes a bug).
+
+The web UI for a specific kernel(say [upstream linux](https://syzkaller.appspot.com/upstream)) shows the `Bisected` status for all bugs.
+
 <div id="bisection"/>
```

Comment:
There is a bug currently. We use the same ID for 2 divs, and this one is obviously wrong.
But don't remove it, keep the div before syzkaller reproducers section and change the id to "syzkaller-reproducers".
Add another div before the new fix bisection section and use "fix-bisection" id.
These can be used as anchors in links.